src/main/resources/scripts/rwa_2905_check_and_update.sql

```diff
@@ -0,0 +1,42 @@
+-- PRE DEPLOYMENT DATA PREPARATION STEPS
+-----------------------------------------
+select task_id from cft_task_db.tasks where role_category is null and jurisdiction = 'CIVIL';
+--  take a backup of all the task_id's
```

Comment:
👍 
Might have to get this at the time of execution.
Although a very small possibility and possibly negligible risk,  it's not guaranteed that even if the number of nulls are the same as today, that the task_ids are the same as in the list in your rollback.